src/analysis/souffle/taint.dl

```diff
@@ -78,4 +78,13 @@ path(from, to) :- edge(from, intermediate), path(intermediate, to).
 mayHaveTag(tgt, tag) :- claimHasTag(tgt, tag).
 mayHaveTag(tgt, tag) :- edge(src, tgt), mayHaveTag(src, tag).
```

Comment:
You might have to sync your branch to get Andrew's changes.

Reply:
Just did this, thank you for the reminder.